DevelopedCalculator.js

```diff
@@ -11,7 +11,7 @@ class DevelopedCalculator extends Calculator {
     }
 
     onButtonNumberClick_(event) {
-        this.super.history.push(this.currentResult)
+        super.this.history.push(this.currentResult)
```

Comment:
the `history` property which is an array is defined on the `DevelopedCalculator` class
how do you access a property on a class?

do you use `this` or `super` or both? 
or you just do some random thing hopping it will work?

Reply:
then, add the current result to the history array

this.history.push(this.currentResult)

Reply:
this `this.history.push(this.currentResult)` is correct
my question why you're using `super.this` where did you find this?

Reply:
sorry I was thinking you told me to add :
super.history.push(this.currentResult)

and I added what i thought very long time ago

https://github.com/yaaHosein/Calculator1/pull/29#issuecomment-822770187

Reply:
Please read carefully